ReSharper.FSharp/src/Daemon.FSharp/src/FSharpSymbolHighlightingUtil.cs

```diff
@@ -82,6 +98,13 @@ public static string GetHighlightingAttributeId([NotNull] this FSharpSymbol symb
         case FSharpEntity entity when !entity.IsUnresolved:
           return GetEntityHighlightingAttributeId(entity);
 
+        case FSharpParameter parameter: // FSharpParameter is a static member in a generic constraint, not a method parameter
```

Comment:
AFAIK `FSharpParameter` is currently used for named args in method invocations and in type providers instantiations. Function parameters are represented as values reported as `FSharpMemberOrFunctionOrValue` symbols and I'm not sure if there's a way to tell them apart easily via `FSharpSymbol` APIs. Maybe it could be added the same way we've added `IsValCompiledAsMethod` previously.

Reply:
Thank you. This is important info. I guess I will have to add lots of different tests.. :)

Reply:
This code was removed.